vscode-lean4/test/suite/info/info.test.ts

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'assert';
+import { suite } from 'mocha';
+import * as vscode from 'vscode';
+import { initLean4Untitled, assertStringInInfoview, findWord } from '../utils/helpers';
+import { InfoProvider } from '../../../src/infoview';
+
+suite('InfoView Test Suite', () => {
+
+    test('Copy to Comment', async () => {
+
+        console.log('=================== Copy to Comment ===================');
+
+        const lean = await initLean4Untitled('#eval Lean.versionString');
```

Comment:
```suggestion
        const lean = await initLean4Untitled('#eval "copyToCommentTest"');
```
We don't want this test to fail immediately when switching to (or just testing with) release versions.

Reply:
Sure I used numbers instead because I want the string coming from infoview copy to comment to not already be in the text editor.